android/rctmln/src/main/java/com/maplibre/rctmln/components/location/LocationComponentManager.java

```diff
@@ -67,6 +67,15 @@ public void setRenderMode(@RenderMode.Mode int renderMode) {
         }
     }
 
+    public void setPreferredFramesPerSecond(int preferredFramesPerSecond) {
+       if(preferredFramesPerSecond <= 0) {
```

Comment:
Should we also check for a max value?

Reply:
There's no max value for this setting, by default Mapbox sets 3-15 (depending on zoom). IMO we can leave it like that. What do you think?

Reply:
Yup, makes sense 👍